docker-compose-test.yml

```diff
@@ -14,6 +14,7 @@ services:
       - STORYBOOKS_GCP_BUCKET
       - STORYBOOKS_GCP_PRIVATE_KEY_BASE64
       - STORYBOOKS_GCP_CLIENT_EMAIL
+      - STORYBOOKS_USE_YARN_WORKSPACES=false
```

Comment:
Small tweak here to disable a feature used by FxA but not Experimenter. Previously, the subcommand was failing silently and allowing the rest of the process to complete. But now that errors cause the whole run to fail, this subcommand started failing things early.